supplier_app/models.py

```diff
@@ -401,6 +406,27 @@ def __str__(self):
         )
 
 
+class BankAccountArgentina(models.Model):
```

Comment:
If we are making a specific bank account for Argentina we should move all the specific fields like `bank_cbu_file` to this model.

Reply:
bank_info is specific for Argentina as well